### ISSUE TITLE
toggle layers on/off interactively

### ISSUE THREAD

**Issue by joamatab** [solgaardlab/dphox#2]
Hi Sunil, your holoviews plots look really good

an interactive way to toggle layers on/off would be great

what do you think of using something like this?
https://holoviews.org/user_guide/Customizing_Plots.html

**Comment by sunilkpai** [comment]
Yeah, I totally agree with you that would be ideal. I tried to do the simplest possible thing with holoviews first (just plot the device and use `exclude_layers` to mimic this capability). But ultimately this would require some custom plot hooks and buttons to the side of the plot, which shouldn't be too bad. Not sure when I'll get time to implement it, but definitely something I'd love to add :)

**Comment by sunilkpai** [comment]
Actually, this may be one route: https://github.com/holoviz/holoviews/issues/3882
